tensorflow_lite_support/python/task/vision/pybinds/BUILD

```diff
@@ -23,3 +23,20 @@ pybind_extension(
         "@pybind11_protobuf//pybind11_protobuf:native_proto_caster",
     ],
 )
+
+pybind_extension(
+    name = "image_classifier",
+    srcs = [
+        "image_classifier.cc",
+    ],
+    module_name = "image_classifier",
```

Comment:
Please adopt the same conventon as image embedder here, i.e. "_pywrap_image_classifier"